titanic.ipynb

```diff
@@ -43,9 +43,6 @@
         "# Load the Titanic dataset\n",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 Line 5_
```python
import seaborn as sns
import matplotlib.pyplot as plt
import pandas as pd

# Load the Titanic dataset
```
test five

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/26)._